themes/README.md

```diff
@@ -0,0 +1,63 @@
+# Mumble Theme(s)
+
+This directory contains the theme(s) Mumble ships with by default.
+
+A theme consists of the following components:
+- Style definitions (e.g. used colors, border widths, etc.) that are defined as QSS files (Qt's version of CSS)
+- Icons
+- Icon-Mapping that map a given name (alias) that is used in the source code to a particular icon in the theme
+
+The heart of a theme is its corresponding `.qrc` file. This file contains the Icon-Mapping and also a mapping for the QSS files. It is responsible of
```

Comment:
The theme repository / the theme subfolder does not have qrc files though?

Reply:
Yes it does. it's in directly in `themes`. Note that it is _not_ in `themes/Mumble` though. That file always lived in the main repo